src/ActiveLogin.Authentication.BankId.AspNetCore/Areas/BankIdAuthentication/Views/BankId/_LoginScript.cshtml

```diff
@@ -4,8 +4,10 @@
     (function (options) {
         // Pre check
         var statusElement = document.querySelector('#bankIdLoginStatus');
+        var statusCardElement = statusElement.querySelector('.card');
```

Comment:
As the class `card` comes from Bootstrap, I would like to have something more specific for this.
Could you add `status-wrapper` as a class as well? That gives us something unique to look for.

Reply:
Absoulutely. Love the decoupling it gives.